fe/fe-core/src/main/java/com/starrocks/backup/Repository.java

```diff
@@ -80,7 +80,7 @@
 public class Repository implements Writable {
     private static final Logger LOG = LogManager.getLogger(Repository.class);
 
-    public static final String PREFIX_REPO = "__palo_repository_";
+    public static final String PREFIX_REPO = "__starrocks_repository_";
```

Comment:
For compatible reason, this prefix can not be changed easily.
When this is changed, new version should know old version's repo path.

Reply:
@imay if rename and mkdir APIs are implemented in broker, then the old repo path can renamed to the new name.